src/web/components/StickyBottomBanner/BrazeBanner.tsx

```diff
@@ -150,10 +150,24 @@ const getMessageFromBraze = async (
     return canShowPromise;
 };
 
+const FORCE_BRAZE_ALLOWLIST = [
+    'preview.gutools.co.uk',
+    'preview.code.dev-gutools.co.uk',
+    'localhost',
+    'm.thegulocal.com',
+];
+
 const getBrazeMetaFromQueryString = (): Meta | null => {
     if (URLSearchParams) {
-        const params = new URLSearchParams(window.location.search);
         const qsArg = 'force-braze-message';
+
+        if (!FORCE_BRAZE_ALLOWLIST.includes(window.location.hostname)) {
```

Comment:
Maybe use `FORCE_BRAZE_DENYLIST` to remove the negation?

Reply:
Probably safer to allow specifics, than deny specifics?

Reply:
Yeah that was my thinking, that allowing specifics makes more sense.